src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration debug="false">
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    <property name="LOG_PATTERN"
+              value="%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr(${PID:- }){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m | %mdc %n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}"/>
+
+    <springProperty scope="context" name="app" source="spring.application.name"/>
+
+    <springProfile name="!logstash">
+        <appender name="consoleAppender" class="ch.qos.logback.core.ConsoleAppender">
+            <layout class="ch.qos.logback.classic.PatternLayout">
+                <Pattern>${LOG_PATTERN}</Pattern>
+            </layout>
+        </appender>
+        <appender name="aiAppender" class="ch.qos.logback.core.helpers.NOPAppender"/>
+    </springProfile>
+
+    <springProfile name="logstash">
```

Comment:
`prison-offender-search` (and the other jvm apps I’ve looked at) are using logstash by default and disabling it with either the `stdout` or `dev` spring profiles.   Is it best we follow that pattern?

Reply:
Possibly, I initially used [this](https://github.com/ministryofjustice/prisoner-offender-search/blob/2f2de8e6f30a44ca6c11d146cd13b105e83241fb/src/main/resources/logback-spring.xml) as my starting point but then stopped because honestly the relationship between `logAppender` and `consoleAppender` in this config doesn't make any sense to me. [This configuration](https://github.com/ministryofjustice/offender-assessments-api/blob/main/src/main/resources/logback-spring.xml) seems saner but I'm happy to go with dev/stdout as the profiles if you prefer?

Reply:
I only raised it for consistency. After looking at both implementations I find yours easier to understand so I'm happy to go with this

Reply:
@markberridge I've been trying to work out what the 'profile' actually does and even `logstash` is a bit misleading as that's juts a part and its just the tech that's being used. The best I've come up with is `instrumented` which is descriptive and I think accurate but not the easiest to type 🤔

Reply:
as an aside - I've been trying to keep it so that the default is to log to stdout and it 'just works' ™️. Would rather not break existing run configurations that don't have a profile set on local machines for example